lib/bolt/cli.rb

```diff
@@ -837,11 +840,13 @@ def add_project_module(name, project)
     def assert_project_file(project)
```

Comment:
Huh! This is actually dead code, because if the user doesn't have a projectfile this will be raised:
```
Unable to use command 'bolt module install' when 'modules' is not configured in bolt-project.yaml. Use 'bolt puppetfile install' instead.
```
From line 202 first. I think I'd move `assert_puppfile_or_module` to be called in the `puppetfile` and `module` handlers, maybe even just after `assert_projectfile`, but you might have better ideas of how that flow should work.